upgrades/pre/upgrade-2-12-0.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -eu
+
+if [[ $(helm status -n external-secrets external-secrets-artifacts 2>/dev/null) ]]; then
+  helm uninstall -n external-secrets external-secrets-artifacts
```

Comment:
Do we also need to uninstall te following?:
```
external-secrets
vault-operator
vault-operator-artifacts
```
These helm charts were still installed after upgrading is that correct?